brian_cheung/week_two_challenges/kanye_records.md

```diff
@@ -19,13 +19,13 @@ kanyeRecords = [
 1. Print the name of every Kanye record.
 
 ```ruby
-kanyeRecords.each do |hash_iterate| puts hash_iterate[:title] end
+kanyeRecords.each { |hash_iterate| puts hash_iterate[:title] }
 ```
```

Comment:
This `hash_iterate` value doesn't have the best name. What is "each" thing here that has a title?